MAUI/ComboBox/UI-Customization.md

```diff
@@ -173,4 +173,528 @@ comboBox.MaxDropDownHeight = 150;
+            </DataTemplate>
+        </editors:SfComboBox.ItemTemplate>
+    </editors:SfComboBox>
+
+{% endhighlight %}
+{% highlight C# %}
+
+    EmployeeViewModel employee = new EmployeeViewModel();
+
+    SfComboBox comboBox = new SfComboBox()
+        {
+            HeightRequest = 34,
+            WidthRequest = 280,
+            BindingContext = employee,
+            ItemsSource = employee.Employees,
+            DisplayMemberPath = "Name",
+            Placeholder = "Enter an employee",
+            TextMemberPath = "Name",
+        };
+
+    DataTemplate itemTemplate = new DataTemplate(() =>
+    {
+        Grid grid = new();
+        grid.Margin = new Thickness(0, 5);
+        grid.HorizontalOptions = LayoutOptions.Center;
+        grid.VerticalOptions = LayoutOptions.Center;
+        ColumnDefinition colDef1 = new ColumnDefinition() { Width = 48 };
+        ColumnDefinition colDef2 = new ColumnDefinition() { Width = 220 };
+        RowDefinition rowDef = new RowDefinition() { Height = 50 };
+        grid.ColumnDefinitions.Add(colDef1);
+        grid.ColumnDefinitions.Add(colDef2);
+        grid.RowDefinitions.Add(rowDef);
+
+        Image image = new();
+        image.HorizontalOptions = LayoutOptions.Center;
+        image.VerticalOptions = LayoutOptions.Center;
+        image.Aspect = Aspect.AspectFit;
+        image.SetBinding(Image.SourceProperty, ("ProfilePicture"));
+        Grid.SetColumn(image, 0);
+
+        StackLayout stack = new();
+        stack.Orientation = StackOrientation.Vertical;
+        stack.Margin = new Thickness(15, 0,0,0);
+        stack.HorizontalOptions = LayoutOptions.Start;
+        stack.VerticalOptions = LayoutOptions.Center;
+        Grid.SetColumn(stack, 1);
+
+        Label label = new();
+        label.SetBinding(Label.TextProperty, "Name");
+        label.FontSize = 14;
+        label.VerticalOptions = LayoutOptions.Center;
+        label.HorizontalTextAlignment = TextAlignment.Start;
+        label.Opacity = .87;
+
+        Label label1 = new();
+        label1.SetBinding(Label.TextProperty, "Designation");
+        label1.FontSize = 12;
+        label1.VerticalOptions = LayoutOptions.Center;
+        label1.HorizontalTextAlignment = TextAlignment.Start;
+        label1.Opacity = .54;
+
+        stack.Children.Add(label);
+        stack.Children.Add(label1);
+
+        grid.Children.Add(image);
+        grid.Children.Add(stack);
+
+        return new ViewCell { View = grid };
+    });
+    comboBox.ItemTemplate = itemTemplate;
+
+    this.Content = comboBox;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox ItemTemplate](Images/UICustomization/ItemTemplate.png)
+
+## Customize the DropDown (suggestion) item based on condition
+
+The [ItemTemplate]() property helps you to decorate drop-down items conditionally based on their content using the custom templates. The default value of `ItemTemplate` is `null`.
```

Comment:
The [ItemTemplate]() property helps you to decorate drop-down items conditionally based on their content using the custom templates. The default value of the `ItemTemplate` is `null.`